lua/gatekeeper/staticlib.lua

```diff
@@ -421,8 +425,8 @@ ffi.cdef[[
 void rte_log_set_global_level(uint32_t log_level);
 uint32_t rte_log_get_global_level(void);
 
-int rte_log_set_level(uint32_t type, uint32_t level);
-int rte_log_get_level(uint32_t type);
```

Comment:
These functions are still useful for debugging DPDK and should be preserved. Group them with the two functions above, and add the comment below to them clarify their purpose.
`/* These functions are meant to help one to investigate issues that depend on the internals of DPDK. */`
Break this comment into as many lines as needed.